src/ServicePulse.Host/vue/src/router/index.ts

```diff
@@ -1,22 +1,33 @@
 import { createRouter, createWebHashHistory, type RouteRecordRaw } from "vue-router";
-import config from "./config";
+import config, { type RouteItem } from "./config";
 
 function meta(item: { title: string }) {
   return { title: `${item.title} • ServicePulse` };
 }
 
-const routes = config.map((item) => {
-  return {
+const routes = config.flatMap<RouteRecordRaw>((item) => {
+  const result: RouteRecordRaw[] = [];
+  result.push({
     path: item.path,
+    name: item.path,
     meta: meta(item),
-    alias: item.alias ?? [],
```

Comment:
I found an issue with using aliases vs redirects. This was introduced in #1728.
The summary, is that redirects work better, so what I have done here is create redirects from aliases.